src/editor/CSSInlineEditor.js

```diff
@@ -151,6 +151,27 @@ define(function (require, exports, module) {
         return html;
     }
 
+    htmlToCSSProvider.queryProvider = function(hostEditor, pos) {
```

Comment:
Move this below the definition of `htmlToCSSProvider`, it makes it easier to understand.

Also, please add a comment above it:

```js
// XXXBramble: we extend providers so that we can see if one exists without invoking.
```